ckan/logic/action/get.py

```diff
@@ -1889,18 +1889,34 @@ def package_search(context, data_dict):
 
         count = query.count
         facets = query.facets
+        raw_solr_results = {
+            'results': query.results,
+            'highlighting': query.highlighting,
+            'count': query.count,
+            'facets': query.facets,
+        }
     else:
         count = 0
         facets = {}
         results = []
+        raw_solr_results = {}
 
     search_results = {
         'count': count,
         'facets': facets,
         'results': results,
-        'sort': data_dict['sort']
+        'sort': data_dict['sort'],
     }
 
+    include_raw_solr_results = False
+    for item in plugins.PluginImplementations(plugins.IPackageController):
+        if 'include_raw_solr_results' in dir(item):
```

Comment:
This is only needed because I haven't added fallback implementation to IPackageController yet